src/plugins/i18n/locales/en.json

```diff
@@ -621,7 +621,7 @@
         },
         "spotify": {
           "link": {
-            "description": "To connect your Spotify account please follow [a link=guide]this guide[/a]."
+            "description": "To connect to your Spotify account please follow [a link=guide]this guide[/a]."
```

Comment:
Not *really* necessary. `Connecting your Spotify account` sounds more concise and direct than `Connecting to your Spotify account`.